profiles/lib/guidance.lua

```diff
@@ -59,6 +59,14 @@ function Guidance.set_classification (highway, result)
     else
         result.road_classification.may_be_ignored = true;
     end
+
+    lane_count = input_way:get_value_by_key("lanes")
```

Comment:
The width of the road is the only important part. But I've added the missing forward/backward stuff in.